docs/docs/cmd/yammer/yammer-search.md

```diff
@@ -40,7 +40,7 @@ m365 yammer search [options]
 
 ## Examples
 
-Returns the search result summary for the query `community`
+Returns the search result for the query `community`
 
 ```sh
 m365 yammer search --search "community"
```

Comment:
If the `--show` option is required, as designated by `<>`, we should include it here, or running the command will fail

Reply:
Have fixed it in the documentation. It was changed to optional